reproducer-clean.yml

```diff
@@ -44,6 +51,14 @@
         - reproducer-network-env
         - artifacts/run-id
 
+    - name: Ensure sushy_driver key is removed
+      tags:
+        - deepscrub
+      ansible.builtin.lineinfile:
+        path: "{{ ansible_user_dir }}/.ssh/authorized_keys"
+        state: absent
+        regexp: ".*cifmw-sushy_driver$"
```

Comment:
Can we rename the comment to `cifmw-sushy_emulator` to match the role name?
AFICT, `sushy_driver` is not a term used anywhere before this change.